test/validators.test.js

```diff
@@ -12827,6 +12827,16 @@ describe('Validators', () => {
         '19640823-32333',
         '1964082332333'],
     });
+    test({
+      validator: 'isTaxID',
+      args: ['uk-UA'],
+      valid: [
```

Comment:
Doing this from a quick look, so might not be complete; you should be able to increase the coverage by also checking for a number that has 0 as 10th digit (checks for digits[9] === 0)

Reply:
Thanks, fixed it :)